misc/python/materialize/buildkite_insights/costs/extract_stats.py

```diff
@@ -68,7 +73,8 @@ def main() -> None:
                     cost = instance_cost[metadata.removeprefix("aws:instance-type=")]
                     break
             else:
-                raise ValueError("jo instance type found")
+                # Can't calculate cost for mac-aarch64
+                cost = 0
```

Comment:
Shall we print a warning in this case? (optional)

Reply:
I don't think a warning makes sense. For context: The mac-aarch64 queue is fixed, some servers are just around.